internal/controller/pgcluster/pgclustercontroller.go

```diff
@@ -78,6 +80,17 @@ func (c *Controller) waitForShutdown(stopCh <-chan struct{}) {
 	log.Debug("pgcluster Contoller: received stop signal, worker queue told to shutdown")
 }
 
+func getNewReplicaObject(cluster *crv1.Pgcluster, index int) crv1.Pgreplica {
+	newReplica := crv1.Pgreplica{}
```

Comment:
we can specify all fields directly, without filling one by once, since we don't have any functions, which could return error